### DISCUSSION TITLE
2024 Contribution

### DISCUSSION THREAD

**Discussion by Lily-kwamboka** [ModECI/MDF#504] [General]
Hi I am Lilian Kwamboka , I am a final year student  at the University of Eldoret, Kenya taking a bachelor in Computer Science. I am well equipped in Python, Machine Learning and I have experience in development of User Interfaces. I am interested in making a contribution to this project for the May to August 2024 interns.

**Comment by pgleeson** [comment]
Welcome @39148127! See https://github.com/ModECI/MDF/discussions/509.
